tests/cli/test_cli_set_microbial_sample.py

```diff
@@ -78,6 +78,38 @@ def test_valid_application(invoke_cli, disk_store: Store):
     assert signature in disk_store.MicrobialSample.query.first().comment
 
 
+def test_invalid_priority(invoke_cli, disk_store: Store):
+    # GIVEN a database with a sample
+    sample = add_microbial_sample_and_order(disk_store)
+    priority = 'dummy_priority'
+    assert disk_store.MicrobialSample.query.first().priority != priority
+
+    # WHEN calling set sample with an invalid priority
+    result = invoke_cli(['--database', disk_store.uri, 'set', 'microbial-sample',
+                         sample.internal_id, 'sign', '-p', priority])
```

Comment:
Could you move the 'sign' into a variable `signature`? I had no idea what this was until I read what `test_valid_priority` did